content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Stronger together: FAIR-EASE and EuroScienceGateway join forces" 
+tease: "Presenting some outcomes from the first joined FAIR-EASE / EuroScienceGateway workshop"
+authors: "Marie Jossé"
+source_blog: "https://eosc.eu/news/stronger-together-fair-ease-and-eurosciencegateway-join-forces"
+date: "2023-05-22"
+---
+
+# Context
+
+![EOSC EuroScienceGateway](images/eurosciencegateway.png)
+![EOSC Fair-Ease](images/fair_ease_colour.png)
+
+Fair-Ease project aims at providing users with an easy and FAIR tool for discovery and access to environmental multidisciplinary and aggregated data-sets, as well as set up a virtual research environment (EAL - Earth Analytical Lab) through web-based interfaces.
+
+EuroScienceGateway leverage a distributed computing network and user-friendly web portal, facilitating access to compute and storage infrastructures across Europe as well as to data, tools, workflows and services that can be customised to suit researchers’ needs.
+
+Thus, the collaboration of those two EOSC projects will allow the creation of cross-discipline workflows between climate science, biodiversity studies and the earth sciences, creating, sharing and re-using tools and workflows in Galaxy.
+
+
+# The 2-days training session
+
+With the strong support and help of Björn Grüning and Tunc Tkayikci we conducted a Galaxy discovery of 2 days in Lille. 
+
+The first day aimed at providing training on how to use the Galaxy platform. In order to do so our guiding thread was the climate related tutorials from the GTN written by Anne Fouilloux (https://training.galaxyproject.org/training-material/topics/climate/tutorials/pangeo-notebook/tutorial.html, https://training.galaxyproject.org/training-material/topics/climate/tutorials/pangeo/tutorial.html and https://training.galaxyproject.org/training-material/topics/climate/tutorials/panoply/tutorial.html) after a bief introduction from the Galaxy for Ecology initiative. 
```

Comment:
these could be proper links, perhaps

Reply:
I'm not sure I understand sorry, what do you mean by proper links ?

Reply:
I assume something like this.

```suggestion
The first day aimed at providing training on how to use the Galaxy platform. In order to do so our guiding thread was the climate related tutorials from the GTN written by Anne Fouilloux ([Pangeo Notebook in Galaxy](https://training.galaxyproject.org/training-material/topics/climate/tutorials/pangeo-notebook/tutorial.html), [Pangeo ecosystem 101 for everyone](https://training.galaxyproject.org/training-material/topics/climate/tutorials/pangeo/tutorial.html) and [Visualize Climate data with Panoply netCDF viewer](https://training.galaxyproject.org/training-material/topics/climate/tutorials/panoply/tutorial.html)) after a bief introduction from the Galaxy for Ecology initiative. 
```

It would render as shown below.

The first day aimed at providing training on how to use the Galaxy platform. In order to do so our guiding thread was the climate related tutorials from the GTN written by Anne Fouilloux ([Pangeo Notebook in Galaxy](https://training.galaxyproject.org/training-material/topics/climate/tutorials/pangeo-notebook/tutorial.html), [Pangeo ecosystem 101 for everyone](https://training.galaxyproject.org/training-material/topics/climate/tutorials/pangeo/tutorial.html) and [Visualize Climate data with Panoply netCDF viewer](https://training.galaxyproject.org/training-material/topics/climate/tutorials/panoply/tutorial.html)) after a bief introduction from the Galaxy for Ecology initiative.